README.md

```diff
@@ -2,6 +2,8 @@
 
 [![Build Status](https://travis-ci.org/ozee31/cakephp-cors.svg?branch=master)](https://travis-ci.org/ozee31/cakephp-cors)
 
+It's a pull request from [ozee31/cakephp-cors](https://github.com/ozee31/cakephp-cors).
```

Comment:
Could you please remove this change from PR?

This lines will be unnecessary once it will be merged on ozee31/cakephp-cors;